<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>390</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="">
     <attribute name="title">
      <string>로그인</string>
     </attribute>
     <widget class="QLabel" name="idlabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="pwlbl">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>PW</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="pw">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>로그인</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>회원가입</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>57</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>PW</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>85</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>이름</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>112</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>이메일</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>나이</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="joinid_edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="joinpw_edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="joinname_edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="joinemail_edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="joinage_edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="join_btn">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>회원가입</string>
      </property>
     </widget>
     <widget class="QPushButton" name="joinreset_btn">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
     <widget class="QPushButton" name="idcheck_btn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ID check</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>회원조회</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>이메일</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>PW</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>이름</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>나이</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>회원 조회</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>정보수정</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>회원 관리 프로그램</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
